Memory Game/GameApp/app.js

```diff
@@ -0,0 +1,123 @@
+const cards = document.querySelectorAll('.card');
+const container = document.querySelector('.container');
+cardsArray = Array.from(cards);
```

Comment:
I think this should be 
```suggestion
const cardsArray = Array.from(cards);
```

Actually, I don't really see a reason to convert the node list to an array since you use `forEach` but maybe I missed something.